src/App.tsx

```diff
@@ -1,223 +1,237 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useState,
+  useMemo,
+  useContext,
+  useEffect,
+  useRef,
+} from 'react';
 import { AuthContext } from './components/Auth/AuthContext';
+import { Todo } from './types/Todo';
+import {
+  addNewTodo,
+  deleteTodo,
+  getTodos,
+  updateTodoTitle,
+  updateTodoStatus,
+} from './api/todos';
+import { FilterType } from './types/FilterType';
+import { Error } from './types/Error';
+import { Header } from './components/Header/Header';
+import { Footer } from './components/Footer/Footer';
+import { TodoList } from './components/TodoList/TodoList';
+import
+{ ErrorNotification } from './components/ErrorNotification/ErrorNotification';
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState<FilterType>(FilterType.All);
+  const [isAdding, setIsAdding] = useState(false);
+  const [deletedTodoIDs, setDeletedTodoIDs] = useState<number[]>([]);
+  const [errorMessage, setErrorMessage] = useState<Error>({
+    hasMessage: '',
+  });
+
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
 
+  const getTodosFromServer = async (userId: number) => {
+    try {
+      const todosFromServer = await getTodos(userId);
+
+      setTodos(todosFromServer);
+    } catch (error) {
```

Comment:
```suggestion
    } catch {
```
Can be skipped, if unused. Fix everywhere.